app/models/category.rb

```diff
@@ -0,0 +1,7 @@
+class Category < ApplicationRecord
+  belongs_to :user
+  has_many :expense_categories, dependent: :destroy
+  has_many :expenses, through: :expense_categories, dependent: :destroy
+
+  validates :name, presence: true
+end
```

Comment:
- [ ] Kindly change this association to match the ERD guidelines, a category should belong to author like so `belongs_to :author, class_name: User`. This change should be reflected in the scheme. Categories table should have an `author_id` instead of `user_id`.